### PR TITLE
admin/cleanup

### PR DESCRIPTION
**Pull request by evamaxfield**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.

Various documentation improvements and other bug fixes for the repo.
This also now adds an optional step `raw` that can include the raw data in the quilt package. More work towards a single system to push our integrated cell dataset to quilt. (This definitely doesn't overlap with `cvapipe` :upside_down_face:)

Files here: https://staging.allencellquilt.org/b/allencell-internal-quilt/tree/aics/actk/admin.cleanup/

- [x] Provide relevant tests for your feature or bug fix.
- [x] Provide or update documentation for any feature added by your pull request.

There will be one more update to this before 0.2.0 is released to resolve the new changes in the LabKey database / the segmentation files. See discussion here: https://github.com/aics-int/cvapipe/issues/7#issuecomment-676808610

Thanks for contributing!

### CODE REVIEW

**Review by ritvikvasan — APPROVED** [review]
Looks good to me.

File: .gitignore [hunk 1]
```diff
@@ -111,6 +111,9 @@ ENV/
 # mkdocs documentation
 /site
 
+# VSCode
+.vscode/
```
Comment (donovanr) [inline .gitignore:115]: nice

File: README.md [hunk 2]
```diff
@@ -44,8 +44,44 @@ also for the entire pipeline with:
 actk all run --dataset {path to dataset} --cell_ceiling_adjustment {integer}
 ```
 
-To run in distributed mode across the SLURM cluster at AICS you can add the
-`--distributed` flag to the pipeline call.
+See the [steps module in our documentation](https://AllenCellModeling.github.io/actk/actk.steps.html)
+for a full list of parameters for each step
+
+#### Pipeline Config
```
Comment (donovanr) [inline README.md:50]: this is awesome

File: actk/bin/all.py [hunk 3]
```diff
@@ -43,6 +43,7 @@ def __init__(self):
     def run(
         self,
         dataset: str,
+        include_raw: bool = False,
```
Comment (donovanr) [inline actk/bin/all.py:47]: raw here means full sized fovs right?

File: actk/steps/raw/raw.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            metadata_columns=metadata_columns,
+            **kwargs,
+        )
+
+    @log_run_params
+    def run(self, dataset: Union[str, Path, pd.DataFrame, dd.DataFrame], **kwargs):
+        """
+        Simple passthrough to store the dataset in local_staging/raw.
+        This does not copy any the image files to local_staging/raw, only the manifest.
+        This is an optional step that will only run if you want to upload the raw data.
```
Comment (donovanr) [inline actk/steps/raw/raw.py:51]: have you tried actually pushing the upload to see if the raw data gets into quilt ok?

File: actk/steps/single_cell_images/single_cell_images.py [hunk 5]
```diff
@@ -277,7 +279,9 @@ def run(
             Default: 95.0
 
         bbox: tuple, list, dict
-        Hard coded ZYX dimensions to set the bounding box. For example, (64, 168, 104)
+            Hard coded ZYX dimensions to set the bounding box.
+            Note: This overrides the `bounding_box_percentile` parameter.
```
Comment (donovanr) [inline actk/steps/single_cell_images/single_cell_images.py:283]: great

File: docs/dataset_fields.md [hunk 6]
```diff
@@ -2,17 +2,87 @@
...
-| ChannelIndexStructure        	| The integer index of the structure channel in a raw microscopy image after it has been read into memory.                                            	| 0, 1, 2, 3                                                    	| None  	|
-| ChannelIndexBrightfield      	| The integer index of the brightfield channel in a raw microscopy image after it has been read into memory.                                          	| 0, 1, 2, 3                                                    	| None  	|
-| StandardizedFOVPath          	| The path to read a standardized FOV image file. This file is generated from the "StandardizeFOVArray" step.                                         	| /allen/aics/modeling/jacksonb/data/example_fov_array.ome.tiff 	| None  	|
-| CellFeaturesPath             	| The path to read a cell features JSON file. This file is generated from the "SingleCellFeatures" step.                                              	| /allen/aics/modeling/jacksonb/data/example_cell_feats.json    	| None  	|
+### CellId
```
Comment (donovanr) [inline docs/dataset_fields.md:5]: this is nicer

**Review by donovanr — APPROVED** [review]

File: actk/bin/all.py (hunk 3 above)
Comment (evamaxfield) [reply actk/bin/all.py:47]: Correct. The source microscopy images.

File: actk/steps/raw/raw.py (hunk 4 above)
Comment (evamaxfield) [reply actk/steps/raw/raw.py:51]: Check the files pushed. They arrive just fine :)

**Pull request merged**